akka-actor-typed/src/main/scala/akka/actor/typed/internal/ActorMdc.scala

```diff
@@ -11,17 +11,20 @@ import org.slf4j.MDC
  * INTERNAL API
  */
 @InternalApi private[akka] object ActorMdc {
-  val SourceKey = "akkaSource"
-  val TagsKey = "akkaTags"
+  val SourceActorSystemKey = "sourceActorSystem"
```

Comment:
the reason for the naming inconsistency is that this was an already existing mdc entry

Reply:
And we're not referring to the constants in `akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala` since the logging is supposed to be independent of the akka.event logging, even though it uses consistent keys. Makes sense to me :+1: . Might be good to introduce an `akkaSystemName` key or so so that we can eventually deprecate/drop the inconsistent `sourceActorSystem`? Not for this PR though.